src/node/utils/cssUtils.ts

```diff
@@ -169,3 +149,35 @@ export function getCssImportBoundaries(
   }
   return boundaries
 }
+
+const syntax: Record<string, Syntax> = {
+  sass: require('postcss-sass'),
+  scss: require('postcss-scss'),
+  less: require('postcss-less'),
+  styl: require('postcss-styl'),
+  stylus: require('postcss-styl')
+}
+
+export async function parseCssImport(
+  css: string,
+  filePath: string,
+  preprocessLang: string
+) {
+  if (!css.includes('@import')) return
+
+  // Todo: can be handle with preprocess parser, that's only parse once
+  const { messages } = await postcss([require('postcss-import')()]).process(
+    css,
+    { syntax: syntax[preprocessLang], from: filePath }
+  )
```

Comment:
=.= Here i use postcss parser when has css preprocess lang. It will can be work and it is convenient and has expand for process each preprocess lang with css.

Reply:
This results in two postcss calls for all CSS - which is less than ideal. It might be worth it for pre-processors to avoid dealing with their different import dependency APIs, but we should avoid the double-processing for plain CSS and only call it if `preprocessLang` actually exists. We already have `postcss-import` in `compileCss`.